### DISCUSSION TITLE
Install StarShip in home folder?

### DISCUSSION THREAD

**Discussion by Beinish** [starship/starship#5145] [Q&A] [answered]
Hello, I was wondering if it's possible to install StarShip in my home folder instead of the default `/usr/local/bin`. I have a network mounted home folder and I jump through a lot of servers daily, it would be nice to see the same shell everywhere :)

Thanks

**Comment by mickimnet** [comment]
Why don't you just move the _Starship_ binary to the home folder? (Sidenote: you're aware, that it will only work if the different servers have the same operating system)
Else wise, the installation method is based on the operating system and the package manager you are using.

**Comment by davidkna** [comment, marked as answer]
You can put the binary wherever you want. If you are using the official installation script, it has a flag to change the binary installation directory, e.g: `curl -sS https://starship.rs/install.sh | sh -s -- -b "$HOME/.local/bin"`
